config/reliefweb_import.plugin.importer.worldbank.yml

```diff
@@ -0,0 +1,19 @@
+enabled: true
+max_import_attempts: 3
+classification:
+  enabled: true
```

Comment:
Do we need the AI classification? Isn't the data from the API good enough?

Reply:
Discussed with Mary, using AI for theme and content format